lib/simple_config.py

```
@@ -208,20 +208,17 @@ def read_user_config(path):
     if not path:
         return {}
     config_path = os.path.join(path, "config")
-    try:
-        with open(config_path, "r") as f:
-            data = f.read()
-    except IOError:
-        #print_msg("Error: Cannot read config file.", path)
+    if not os.path.exists(config_path):
         return {}
+    with open(config_path, "r") as f:
+        data = f.read()
     try:
         result = json.loads(data)
-    except:
+    except Exception:
         try:
             result = ast.literal_eval(data)
-        except:
-            #print_msg("Error: Cannot read config file.")
-            return {}
+        except Exception:
+            raise Exception('Failed to parse config file {}'.format(config_path))
```

I think it is insane that electrum / lbryum simple swallow errors while reading the config file.

This should be a separate commit though.